src/pages/who-we-are/index.tsx

```diff
@@ -35,8 +37,66 @@ const EndSeparator = styled.div`
         border-bottom: 40px solid transparent;
     }
 `
-
 const AboutUs = () => {
+    const target1 = '#simti'
```

Comment:
As a suggestion you can move the `options`, `lazy_components` and `lazyTemplate` outside the component!